common/src/main/res/values/strings.xml

```diff
@@ -256,13 +256,19 @@
     <string name="screen_orientation_option_value_landscape">landscape</string>
     <string name="get_help">Get Help</string>
     <string name="grant_permission">Grant Permission</string>
+    <string name="header_name_1">Name of the 1. header</string>
+    <string name="header_name_2">Name of the 2. header</string>
+    <string name="header_value_1">Value of the 1. header</string>
+    <string name="header_value_2">Value of the 2. header</string>
```

Comment:
Ordinals in English are written like "1st", "2nd", etc. To make this easier for translation, I suggest just using 1, 2, etc.

```suggestion
    <string name="header_name_1">Custom Header 1 Name</string>
    <string name="header_name_2">Custom Header 2 Name</string>
    <string name="header_value_1">Custom Header 1 Value</string>
    <string name="header_value_2">Custom Header 2 Value</string>
```

Reply:
Changed, thanks

Reply:
@Meister1977 it seems this fix got reverted